fuzzing/private/fuzz_test.bzl

```diff
@@ -230,6 +231,10 @@ def java_fuzz_test(
           binary rule.
     """
 
+    if data != None:
+        fail("java_fuzz_test does not support the 'data' attribute, use " +
+             "'resources' instead")
```

Comment:
Nit: Instead of defining a new attribute to the signature of the macro, consider calling `binary_kwargs.pop()` and checking the returned value.

Reply:
That's good advice, thanks!